com.solvespace.SolveSpace.json

```diff
@@ -0,0 +1,150 @@
+{
+    "$schema": "https://raw.githubusercontent.com/TingPing/flatpak-manifest-schema/master/flatpak-manifest.schema",
```

Comment:
what is this?

Reply:
ah, oops, didn't mean to commit that. It's a JSON manifest to get in-editor help.